lib/server.rb

```diff
@@ -0,0 +1,80 @@
+require 'data_mapper'
+require 'sinatra/base'
+require 'rack-flash'
+
+env = ENV['RACK_ENV'] || 'development'
+
+DataMapper.setup(:default, "postgres://localhost/bookmark_manager_#{env}")
```

Comment:
ideally database setup would be extracted to a setup file that was then required here